wemake_python_styleguide/checker.py

```diff
@@ -118,6 +123,23 @@ def __init__(
         self.tree = transform(tree)
         self.filename = filename
         self.file_tokens = file_tokens
+        self.cst = self.build_cst()
+
+    def read_py3(self) -> str:
+        """Read the source code."""
+        with tokenize.open(self.filename) as src_code:
+            return src_code.read()
+
+    def build_cst(self) -> libcst.Module:
```

Comment:
Can and should we cache this call?

Reply:
hm, I can do that
```
@lru_cache(None)
def build_cst(self, filename) -> libcst.Module: ...
```
but checker is created for every files and I didn't found that flake8 cached ast.(only use git for find diff) What do you think about it?

Reply:
So, we call this function once. Ok, no action is required. Thanks!